legend-engine-xts-snowflakeApp/legend-engine-xt-snowflakeApp-api/src/main/java/org/finos/legend/engine/language/snowflakeApp/deployment/SnowflakeAppArtifactGenerationExtension.java

```diff
@@ -38,15 +40,25 @@ public String getKey()
     public boolean canGenerate(PackageableElement element)
     {
         return false;
-       // return element instanceof Root_meta_external_function_activator_snowflakeApp_SnowflakeApp;
     }
 
 
     @Override
     public List<Artifact> generate(PackageableElement element, PureModel pureModel, PureModelContextData data, String clientVersion)
     {
-        return null;
+       try
+        {
+            /* add content logic here */
+            Artifact output = new Artifact(null, element.getName() + "_snowflakeAppArtifact.json", "json");
```

Comment:
My understanding is that each artifact is already generated inside a directory for the element.

Hence, I don't thin we should do this concatenation of element name with the suffix, as it will make the name more complex it needs to be.